sql/sql.y

```diff
@@ -74,11 +73,13 @@
 	type trainClause struct {
 		estimator string
 		attrs     attrs
-		columns   exprlist
+		cc   	  columnClause
                 label     string
 		save      string
 	}
 
+	type columnClause map[string]exprlist
```

Comment:
If there is only one column in the column clause, and there is no FOR in that column, how do we handle the case? Are we going to have only one entry in the map[string]exprlist and the key is ""?  If so, please add a comment here to explain.

Reply:
You are right, will add a comment here.